### DISCUSSION TITLE
Preferences for Access

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-system#27] [Q&A]
Hi gang -- I have a question for you all to weigh in on if you can:

Do you guys like having separate URLs for the logs?

I ask because I'm considering consolidating all of the logs into links on the "BirdNET-system Extractions" page. I did that with the live audio stream service. You'll notice the live stream doesn't have its own URL, but is simply a link on the "BirdNET-system Extractions" page.

Let me know if you think I should consolidate the web-log services as links on the "BirdNET-system Extractions" page or whether I ought to keep them as avahi-aliased URLs. Or both!

Thanks!

**Comment by CaiusX** [comment]
Hi Patrick
Would be most useful (for me) to have in one place on the Extractions page.

**Reply by mcguirepr89** [reply]
Thanks! I'll do that for the BirdNET-Lite beta and you can test it out. I'll keep the avahi-aliases in case its helpful for other development.

**Comment by DD4WH** [comment]
> I ask because I'm considering consolidating all of the logs into links on the "BirdNET-system Extractions" page. I did that with the live audio stream service. You'll notice the live stream doesn't have its own URL, but is simply a link on the "BirdNET-system Extractions" page.

Yes, that is more comfortable to have all the links to the logs/stats etc. on one page instead of different URLs !
